src/gt4py/next/program_processors/runners/dace_fieldview/gtir_sdfg.py

```diff
@@ -677,6 +692,21 @@ def _flatten_tuples(
         def make_temps(
             output_data: gtir_builtin_translators.FieldopData,
         ) -> gtir_builtin_translators.FieldopData:
+            """
+            This function will be called while traversing the result of the dataflow
+            of the nested SDFG to setup the intermediate data nodes in the parent SDFG.
+
+            1st if-branch:
+            Transient nodes actually contain some result produced by the dataflow
+            itself, therefore these nodes are changed to non-transient and an output
+            edge will write the result from the nested SDFG to a new intermediate
+            data node in the parent context.
+
+            2nd and 3rd if-branch:
+            Non-transient nodes are just input nodes that are immediately returned
+            by the lambda expression. Therefore, these nodes are already available
+            in the parent context and can be directly accessed there.
```

Comment:
I would put this description as comments.
Because, a docstring is supposed to be viewed without the source code.

Reply:
Good point.